src/oop_ext/foundation/weak_ref.py

```diff
@@ -185,26 +220,26 @@ class WeakMethodProxy(WeakMethodRef):
     arguments. If the referent's object no longer lives, ReferenceError is raised.
     """
 
-    def GetWrappedFunction(self):
+    def GetWrappedFunction(self) -> Any:
```

Comment:
Could it return `Optional[Callable]` (or `Optional[Callable[..., Any]]`)?

Reply:
Done!